apis/userapi/userapi.yaml

```diff
@@ -1799,6 +1891,28 @@ definitions:
             type: string
             description: userId
 
+  MigrateUserReq:
+    description: request body for migrating user
+    type: object
+    properties:
+      request:
+        type: object
+        properties:
+          userId:
+            type: string
+            description: userId of migrating user
+          action:
+            type: string
+            description: accept/reject only
```

Comment:
One of accept, reject; accept reflects intention of the user to be migrated. reject reflects intention of the user to reject the migration offer.

Reply:
done